lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -251,61 +364,246 @@ proc setOptionVars()
     string $optionMayaScenes = uiRes("m_fileDialogFilterTypes.kMayaScenes");
     string $category = "Files/Projects"; // Since some optionVars are shared
 
-    optionVar -init $forceFactorySettings -category $category
-        -iv referenceUseNamespacesDuringFileIO true
-        -iv referenceOptionsUseRenamePrefix false
-        -sv referenceOptionsRenamePrefix ""
-        -iv referenceOptionsLockReference false
-        -iv referenceOptionsDeferReference false
-        -iv referenceOptionsShareDisplayLayers false
-        -iv referenceOptionsShareShaders false
-        -sv referenceOptionsShareRenderLayers ""
-        -iv referenceOptionsSharedReference 1
-        -sv fileReferenceRefLoadSetting "default"
-        -iv fileReferenceReserveNamespaces false
-        ;
-    
-    optionVar -init $forceFactorySettings -category $category
-        -sv defaultFileReferenceType $optionAllFiles    // same default as createGlobalOptVars.mel
-        -iv ("proxyOptionsUseNew"+$action+"Tag") false
-        -sv ("proxyOptions"+$action+"Tag") ""
-        -sv ("proxyOptionsNew"+$action+"Tag") ""
-        ;
-    
-    //   Namespace object name clash method, including "new", "rename","replace" &"reject".
-    //   "new", always create a new namespace and put everything in it
-    //   "rename", when a name clash happens, rename the new object to ****1,****2...
-    //   "replace", replace old object by new one
-    //   "reject", reserve old one and reject new.
-    //   The default option is "new", to ensure not to destroy the file data
-    optionVar -init $forceFactorySettings -category $category
-        -sv namespaceClashNameMethod "new"
-        -sv fileWorkingNamespaceName ""
-        ;
-    
+    int $mjv = `about -majorVersion`;
+    if ($mjv <= 2022)
+    {
```

Comment:
Handle older versions of Maya. The "optionVar --init" flag was added after Maya 2022.